include/vcpkg/base/messages.h

```diff
@@ -764,8 +766,15 @@ namespace vcpkg
     DECLARE_MESSAGE(ExpectedFailOrSkip, (), "", "expected 'fail', 'skip', or 'pass' here");
     DECLARE_MESSAGE(ExpectedPortName, (), "", "expected a port name here");
     DECLARE_MESSAGE(ExpectedTripletName, (), "", "expected a triplet name here");
-    DECLARE_MESSAGE(ExtendedDocumenationAtUrl, (msg::url), "", "Extended documentation available at '{url}'.");
+    DECLARE_MESSAGE(ExtendedDocumentationAtUrl, (msg::url), "", "Extended documentation available at '{url}'.");
     DECLARE_MESSAGE(FailedToExtract, (msg::path), "", "Failed to extract \"{path}\":");
+    DECLARE_MESSAGE(FailedToParseBinParagraph,
+                    (msg::error_msg, msg::value),
+                    "'{error_msg}' is the error message for failing to parse the Binary Paragraph, '{value}' is the "
+                    "Binary Paragraph",
+                    "[sanity check] Failed to parse a serialized binary paragraph.\nPlease open an issue at "
+                    "https://github.com/microsoft/vcpkg, "
+                    "with the following output:\n{error_msg}\nSerialized Binary Paragraph: \n{value}");
```

Comment:
```suggestion
                    "with the following output:\n{error_msg}\nSerialized Binary Paragraph:");
```

(and just `append_raw` at the call site)

Reply:
Why for this instance of a newline and not for others?